libs/blog/newsletter/feature-newsletter/src/lib/newsletter-error/newsletter-error.component.html

```diff
@@ -0,0 +1,27 @@
+<ng-container *transloco="let t; read: 'newsletter.error'">
+  <div class="flex flex-col items-center justify-center text-center">
+    <div class="text-al-pink mb-2 mt-5 aspect-square h-20">
+      <svg
+        fill="currentColor"
+        height="100%"
+        viewBox="0 0 12 12"
+        width="100%"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M10.2427 0.814333L6 5.05733L1.75734 0.814333C1.627 0.684 1.416 0.684 1.286 0.814333L0.814669 1.28567C0.684336 1.416 0.684336 1.627 0.814669 1.757L5.05734 6L0.814669 10.243C0.684669 10.3733 0.684669 10.5843 0.814669 10.7143L1.286 11.1857C1.41634 11.316 1.62734 11.316 1.75734 11.1857L6 6.94267L10.2427 11.1857C10.373 11.316 10.584 11.316 10.714 11.1857L11.1853 10.7143C11.3157 10.584 11.3157 10.373 11.1853 10.243L6.94267 6L11.1853 1.757C11.3153 1.62667 11.3153 1.41567 11.1853 1.28567L10.714 0.814333C10.5837 0.684 10.3727 0.684 10.2427 0.814333Z"
+          fill="currentColor"
+        />
+      </svg>
```

Comment:
I don't think it is a good idea to put here svg tag

Reply:
We can use `al-icon` but as I mentioned, SVG will not be gotten. Corrected